src/VCR/RequestMatcher.php

```diff
@@ -1,102 +0,0 @@
-<?php
-
-namespace VCR;
-
-/**
- * Collection of matcher methods to match two requests.
- */
-class RequestMatcher
```

Comment:
I had to look up ["BC breaks"](http://stackoverflow.com/q/8891005/135101) :-)

You're right. Those methods are public so of course they could be used externally by anyone. We could leave them in indefinitely, and optionally put deprecation warnings.